crates/autopilot/src/database/auction_prices.rs

```diff
@@ -0,0 +1,28 @@
+use {
+    super::Postgres,
+    anyhow::Result,
+    primitive_types::H160,
+    shared::price_estimation::native::from_normalized_price,
+    std::collections::HashMap,
+};
+
+impl Postgres {
+    pub async fn fetch_latest_prices(&self) -> Result<HashMap<H160, f64>> {
+        let _timer = super::Metrics::get()
+            .database_queries
+            .with_label_values(&["fetch_latest_prices"])
+            .start_timer();
+
+        let mut ex = self.pool.begin().await?;
+        Ok(database::auction_prices::fetch_latest_prices(&mut ex)
+            .await?
+            .into_iter()
+            .filter_map(|auction_price| {
+                Some((
+                    H160::from(auction_price.token.0),
+                    from_normalized_price(auction_price.price)?,
```

Comment:
Also sorry to flip-flop a bit on this. It looks like already returning the `f64` price here is what requires us to have the duplicated logic in the `orderbook/database` and `autopilot/database` crates.
This doesn't seem very nice so maybe we should actually simply return the `U256` prices. That way we only need to have the function in the shared `database` crate and do the conversion inside the init function. That would also allow to keep the new conversion function inside the native price module.